picard/track.py

```diff
@@ -220,6 +224,30 @@ def _get_ignored_folksonomy_tags(self):
             tags = [s.strip().lower() for s in ignore_tags.split(',')]
         return tags
 
+    def update_orig_metadata_images(self):
+        class State:
```

Comment:
I just submitted a common function to imagelist.py that works for all cases. I'm afraid it's using local imports which I'm not a fan of, but I found no other way to avoid circular imports.